wyoming_faster_whisper/__main__.py

```diff
@@ -132,8 +132,12 @@ async def main() -> None:
 
 # -----------------------------------------------------------------------------
 
+def run():
+    asyncio.run(main())
+
+
```

Comment:
Just curious, whats the reason for this change?

Reply:
```
wyoming_faster_whisper:__main__.run
```

The entrypoint calls a function, but it cannot be async as far as I know. So we wrap the entrypoint into a sync function, which ultimately makes no difference to people calling `python -m wyoming_faster_whisper`, but allows exposing `wyoming-faster-whisper` via the `PATH` variable.